skglm/tests/test_datafits.py

```diff
@@ -3,12 +3,13 @@
 
 from sklearn.linear_model import HuberRegressor
 from numpy.testing import assert_allclose, assert_array_less
+from statsmodels.discrete.discrete_model import Poisson as PoissonRegressor
```

Comment:
See discussion here: https://github.com/scikit-learn-contrib/skglm/pull/57/files#r990803934

we need a more advanced scheme to avoid imposing statsmodel for every user, yet to make it clear that it's a test dependency

Reply:
i've changed it